docs/packaging-applications/build-servers/tfs-azure-devops/using-octopus-extension/install-octopus-cli-capability.md

```diff
@@ -3,10 +3,91 @@ title: Installing the Octopus CLI as a capability
 description: This guide covers how to add the Octopus CLI as a capability to your Azure DevOps custom build agents.
 ---
 
-There are times when you may want to install the Octopus CLI on a build agent, such as to avoid downloads, opening any firewalls and or changing proxy rules. There are a few ways in which this can be
-achieved. In every option presented here, you must [install the Octopus CLI](/docs/octopus-rest-api/octopus-cli/index.md) so that it's in your current environment path. 
+Tasks in the Octopus extension use the [Octopus CLI](/docs/octopus-rest-api/octopus-cli/index.md) to execute commands with an instance of Octopus. As a result, the Octopus CLI is required to be installed and available on an agent before subsequent tasks run. There are two ways to fulfill this requirement:
 
-## Images and automation
+1. Use the tool installer task, **Octopus CLI Installer** as part of a build pipeline definition
+2. Install the Octopus CLI into a [self-hosted agent](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/agents#install)
 
-Microsoft provide a number of starting points to create your own build agent images which can be modified to include the Octopus CLI as an added capability. This includes [packer images](https://github.com/actions/virtual-environments/tree/main/images) as well as instructions on [running a self-hosted agent in Docker](https://docs.microsoft.com/en-gb/azure/devops/pipelines/agents/docker).
+Using the tool installer task, **Octopus CLI Installer** in the build pipeline definition is suitable for installing the Octopus CLI just-in-time for a build. This is a required task for builds executed on [Microsoft-hosted agents](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted), which do not offer the ability to pre-load custom software. Alternatively, the Octopus CLI may be installed on a self-hosted agent and expressed as a capability. Once configured, a pipeline may express demands of agents to ensure that the Octopus CLI is available when executing builds.
 
+## Using the Octopus CLI Installer
+
+The **Octopus CLI Installer** task downloads and installs the Octopus CLI, making it available to other tasks in a build pipeline definition. It can be added to a definition through the Classic editor of Azure Pipelines or through the YAML pipeline editor.
+
+Currently, the Octopus extension ships two versions of the **Octopus CLI Installer** task; version 4 is provided for backwards-compatibility with older pipeline definitions while version 5 is recommended because it provides additional features.
```

Comment:
```suggestion
Currently, the Octopus extension ships two versions of the **Octopus CLI Installer** task; version 4 is provided for backward compatibility with older pipeline definitions while version 5 is recommended because it offers additional features.
```